pandas/_config/config.py

```diff
@@ -51,15 +51,19 @@
 from collections import namedtuple
 from contextlib import contextmanager
 import re
+from typing import Dict
 import warnings
 
 DeprecatedOption = namedtuple('DeprecatedOption', 'key msg rkey removal_ver')
 RegisteredOption = namedtuple('RegisteredOption',
                               'key defval doc validator cb')
 
-_deprecated_options = {}  # holds deprecated option metdata
-_registered_options = {}  # holds registered option metdata
-_global_config = {}  # holds the current values for registered options
+# holds deprecated option metdata
+_deprecated_options = {}  # type: Dict[str, DeprecatedOption]
+# holds registered option metdata
+_registered_options = {}  # type: Dict[str, RegisteredOption]
+# holds the current values for registered options
+_global_config = {}  # type: Dict[str, str]
```

Comment:
can you add a blank line between the cases. also pls type _registered_keys

Reply:
`# holds registered option metdata` to `# holds registered_keys metdata` right?

Reply:
no there is another option called registered_keys below (so add a type for that one)

Reply:
Ok, got it. I misunderstood what you meant by "type". I will get this done tomorrow too.

Reply:
@jreback all done, take a look now.